runescape-api/src/main/java/net/runelite/rs/api/RSClient.java

```diff
@@ -127,6 +127,16 @@
 	@Import("gameState")
 	int getRSGameState();
 
+	@Import("checkClick")
+	@Override
+	void setCheckClick(boolean checkClick);
+
+	@Import("mouseX2")
+	void setMouseX2(int mouseX2);
```

Comment:
Maybe these should be given better names now?

Reply:
Yeah, i mostly just wanted to avoid PR to internal due to lazyness :d

Reply:
Updated @Adam-